applications/GeoMechanicsApplication/custom_strategies/schemes/geomechanics_time_integration_scheme.hpp

```diff
@@ -331,14 +325,16 @@ class GeoMechanicsTimeIntegrationScheme : public Scheme<TSparseSpace, TDenseSpac
             << rModelPart.GetBufferSize() << std::endl;
     }
 
-    void CheckSolutionStepsData(const Node& r_node, const Variable<double>& variable) const
+    template <class T>
+    void CheckSolutionStepsData(const Node& r_node, const Variable<T>& variable) const
```

Comment:
A minor style issue:
```suggestion
    void CheckSolutionStepsData(const Node& rNode, const Variable<T>& rVariable) const
```

Reply:
Done